### DISCUSSION TITLE
How to ignore missing images?

### DISCUSSION THREAD

**Discussion by MartinKoeditz** [openSUSE/daps#624] [Q&A] [answered]
Hi,

is there a way to ignore missing images for the HTML output? Maybe some switch?

Regards
Martin

**Comment by ghost** [comment, marked as answer]
Hi Martin, unfortunately, I don't know if there is an option for this either. I am also not entirely sure what your use case for this is -- is it for builds of a specific chapter only? Or would you really like to see the little broken-image symbols in your output?

In any case, as a workaround: You can check for missing images (`daps -d [DC-file] list-images-missing`) and simply run touch for the missing ones: `for f in $(daps -d [DC-file] list-images-missing); do touch "images/src/png/$f.png"; done`.

Obviously, that will change your repo and I am not sure you want that to happen.

**Comment by MartinKoeditz** [comment]
> In any case, as a workaround: You can check for missing images (daps -d [DC-file] list-images-missing) and simply run touch for the missing ones: for f in $(daps -d [DC-file] list-images-missing); do touch "images/src/png/$f.png"; done.

Well, that's enough for the moment.

Thank you
